cerberus/tests/test_utils.py

```diff
@@ -13,3 +22,127 @@ class TestLength(models.TextChoices):
         __empty__ = "empty", "Empty"
 
     assert choice_length(TestLength) == 5
+
+
+def test_choice_length():
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (testing):** Duplicate test function `test_choice_length`.

This function appears to be a duplicate of an earlier test function for `choice_length`. Consider removing or renaming it if it serves a different purpose.